env0/resource_template_test.go

```diff
@@ -35,9 +35,11 @@ func TestUnitTemplateResource(t *testing.T) {
 				ErrorRegex: "RetryMeForDestroy.*",
 			},
 		},
-		Type:               "terraform",
+		Type:               "terragrunt",
 		IsGitlabEnterprise: true,
 		TerraformVersion:   "0.12.24",
+		TerragruntVersion:  "0.35.1",
+		TerragruntTfBinary: "opentofu",
```

Comment:
In this case, user has set nothing. Defaults to opentofu when the template is created.